src/core/main.cpp

```diff
@@ -846,13 +846,15 @@ int main( int argc, char * * argv )
 		else if( ConfigManager::inst()->
 				value( "app", "openlastproject" ).toInt() &&
 			!ConfigManager::inst()->
-				recentlyOpenedProjects().isEmpty() )
+				recentlyOpenedProjects().isEmpty() &&
+			!recoveryFilePresent )
```

Comment:
Haha, it'll maybe look weird in some editors that don't have the same spaces for tabs like github does, don't you think?